README.md

```diff
@@ -29,6 +29,41 @@ to the wider research and game developer communities.
 * Visualizing network outputs within the environment
 * Simplified set-up with Docker (Experimental)
 
+## Quick Start Guide
+
+### Get **Unity**
+
+[Download](https://store.unity.com/download) and install Unity, ML-Agents works with versions that are 2017.1 and above. 
+
+### Get Anaconda environment
+
```

Comment:
Anaconda is not required for Linux, Mac

Reply:
I understand Anaconda is not required, but it is relative easier for beginners to use, especially for gamers who don't know python. After download the dmg and install it, they don't need to deal with setting up $PATH in bash_profile file, and are ready to use pip.

Reply:
I am actually not sold on anaconda as a solution here. It is about the same level of complexity as virtual environments, is a fairly large package.

Reply:
Anaconda is easier because most of the people knows how to install a dmg or a exe file, but not as many people knows how to set $PATH or upgrade from Python 2 to Python 3.  Installing the Anaconda Python 3.6 solves this without needing them to mess with this.